### PR TITLE
PP-166 Create print profiles for the new DD core

### PR DESCRIPTION
**Pull request by Andi-ultimaker**
PP-166 Create print profiles for the new DD core
Updated support material profiles to disable their compatibility with the new DD core

Related to PP-166

Needs to be pulled together with the following pull request in the Cura repository: https://github.com/Ultimaker/Cura/pull/12465

### CODE REVIEW

**Review by CoenSchalkwijk — APPROVED** [review]
LGTM

File: generic_pva.xml.fdm_material [hunk 1]
```diff
@@ -128,6 +131,9 @@ Generic PVA profile. The data in this file may not be correct for your specific
             <hotend id="CC 0.6">
                 <setting key="hardware compatible">no</setting>
             </hotend>
+            <hotend id="DD 0.4">
```
Comment (atrebelo) [inline generic_pva.xml.fdm_material:134]: I know this is not the correct location, but the version number `<version>25</version>` should be increased for each of these files.

**Review by atrebelo — CHANGES_REQUESTED** [review]
The version for these files should be updated.

**Comment by Andi-ultimaker** [comment]
> 

Updated version numbers

**Review by CoenSchalkwijk — APPROVED** [review]
Kudo's to @atrebelo for noticing the req. version update :)

**Review by atrebelo — APPROVED** [review]

**Pull request merged**